### PR TITLE
Add description

### PR DESCRIPTION
**Pull request by dv297**
This is my super awesome PR, please +1 it!

### CODE REVIEW

**Comment by edreesjalili** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# project-1
\ No newline at end of file
+# project-1
+
+This is going to be my super awesome project, made by the best developer, Daniel Vu
```
Comment (dv297) [inline README.md]: I don't know about this, maybe it should XYZ

Comment (dv297) [reply README.md]: No, you're crazy, this is fine.

Comment (dv297) [reply README.md]: Fine, whatever.

Comment (edreesjalili) [reply README.md]: 👍 You're crazy

**Comment by dv297** [comment]
💯 🥇

**Pull request merged**